DigitalLearningSolutions.Web/ViewModels/TrackingSystem/CourseSetup/SearchableCourseStatisticsViewModel.cs

```diff
@@ -24,5 +30,14 @@ public SearchableCourseStatisticsViewModel(CourseStatistics courseStatistics)
         public int InProgressCount { get; set; }
         public string CourseName { get; set; }
         public double PassRate { get; set; }
+        public bool HideInLearnerPortal { get; set; }

```

Comment:
I feel like I'm missing something, but is this used anywhere?

Reply:
Good spot. I'd added it that thinking it would be used, got it factored into the Tags property, and then forgot to remove it when it ended up being an unnecessary property. There's a few other properties that are unused, so I'll remove them as well.